samples/torch/src/main/kotlin/HelloTorch.kt

```diff
@@ -0,0 +1,624 @@
+import kotlinx.cinterop.*
+import torch.*
+import platform.posix.*
+
+interface Disposable {
+    fun dispose()
+}
+
+abstract class FloatTensor(val raw: CPointer<THFloatTensor>) : Disposable {
```

Comment:
I'd suggest to split FloatTensor, FloatVector and disposable handling into separate files for better readability.

Reply:
Done!